tests/test_browser.py

```diff
@@ -2107,6 +2107,16 @@ def test_sdl2_image_jpeg(self):
     ]).communicate()
     self.run_browser('page.html', '', '/report_result?600')
 
+  def test_sdl2_image_formats(self):
+    shutil.copyfile(path_from_root('tests', 'screenshot.png'), os.path.join(self.get_dir(), 'screenshot.not'))
```

Comment:
using a confusing suffix is actually no longer needed as of a few days ago, we no longer automatically decode using browser codecs. We require `--use-preload-plugins` be used. So this can just keep the file with the original name with suffix `png`.